ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -337,6 +337,9 @@ function switch_channel() {
 function migrate() {
     echo "Arch migrating"
     cmd="oc adm upgrade --to-multi-arch"
+    if [[ "${ENABLE_OTA_TEST}" == "OCP-60396" ]]; then
```

Comment:
`ENABLE_OTA_TEST` env missed in cucushift-upgrade-arch-migration-ref.yaml

Reply:
good catch!